🌱 Small improvements to tilt (#9936)
kubernetes-sigs/cluster-api: docs/book/src/developer/tilt.md

COPY --from=tilt-helper /go/kubernetes/client/bin/kubectl /usr/bin/kubectl
```

> **kustomize_folder** (String, default=config/default): The folder where the kustomize file for a provider
> are defined; the path is relative to the provider root folder.

Reviewer: ```suggestion
**kustomize_folder** (String, default=config/default): The folder where the kustomize file for a provider
is defined; the path is relative to the provider root folder.
```

> **kustomize_options** (String, default="""): Options to be applied when running kustomize for genereating

Reviewer: ```suggestion
**kustomize_options** (String, default="""): Options to be applied when running kustomize for generating the
```

> yaml manifest for a provider. e.g. `"kustomize_options": { "--load-restrictor=LoadRestrictionsNone" }`

Reviewer: The example doesn't look like an array to me. Shouldn't it be `[]`?

> **apply_provider_yaml** (Bool, default=true): Whether or not apply the provider yaml.

Reviewer: ```suggestion
**apply_provider_yaml** (Bool, default=true): Whether to apply the provider yaml.
```
